src/wallet/ismine.h

```diff
@@ -14,7 +14,27 @@
 class CWallet;
 class CScript;
 
-/** IsMine() return codes */
+/**
+ * IsMine() return codes, which depend on ScriptPubKeyMan implementation.
```

Comment:
17350: we don't have `ScriptPubKeyMan` implemented yet and some codes described below do not exist atm

Reply:
dropped